README.md

```diff
@@ -110,31 +110,87 @@ The module has a small number of common typographic use cases that are available
 }
 ```
 
-For wrappers ([see wrapper section](#wrappers)):
+##### Heading mixins
+
+Headings in o-typography have a specific font-family, and some additional styling depending on the functionality of the heading.
+
+There are two groups of mixins that are available to style headings, one for article headings, and one for headings that support other types of content, which we are calling 'product'.
+
+The following are two examples of many available mixins.
+
+`oTypographyHeadline` will output styles for an article headline.
+
+Usage:
 
 ```scss
-.article__body {
-	@include oTypographyBodyWrapper;
+.my-article-headline {
+	@include oTypographyHeadline();
+}
+```
+
+Output:
+
+```css
+.my-article-headline {
+	font-family: FinancierDisplayWeb, serif;
+	font-size: 32px;
+	line-height: 32px;
+	font-weight: 700;
+	margin-top: 0px;
+	margin-bottom: 28px;
+	color: #33302e;
 }
 ```
 
-Mixins exist for all the same styles as pre-defined classes, named with a camelCased version of the class name.
+`oTypographyProductHeadingLevel1` is an example of the product variant for an `h1`.
+
+Usage:
+```scss
+.my-non-article-headline {
+	@include oTypographyProductHeadingLevel1();
+}
+```
+
+Output:
+```css
+.my-non-article-headline {
+	margin-top: 0px;
+	margin-bottom: 20px;
+	color: #33302e;
+	font-family: MetricWeb, sans-serif;
```

Comment:
Just double checking, you said you were going to rewrite the README?

Reply:
Yeah I'm going to rewrite after branding. In this PR I made bare-bones changes, rearranging a few sections and noting custom fonts.